state/migration_export.go

```diff
@@ -1248,10 +1248,10 @@ func (e *exporter) spaces() error {
 	e.logger.Debugf("read %d spaces", len(spaces))
 
 	for _, space := range spaces {
-		// We do not export the default space because it is created by default
+		// We do not export the alpha space because it is created by default
 		// with the new model. This is OK, because it is immutable.
 		// Any subnets added to the space will still be exported.
-		if space.Name() == network.DefaultSpaceName {
+		if space.Name() == network.AlphaSpaceName {
```

Comment:
I think this should check the ID so we minimise references to the _name_ in non-test code

Reply:
updated